### ISSUE TITLE
FE | Profile | Update Modal (Are you sure?)

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#46935]
## Background

We need to update the text displayed with the "Are you sure?" modal (the modal displayed when editing the profile and moving to a different field without saving or canceling first).

[Here's a link to the new mock-up](https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/v8kxv7J)

> The primary button label should be "Yes, cancel my changes", and close the modal, and bring the user back to read mode without their changes saved
> 
> The secondary button label should be "No, go back to editing" and close the modal and bring the user back to edit mode with their changes in progress.

(Content ticket #46788)

## Tasks
- [ ] Update UI based on sketch file above
- [ ] Conduct Unit tests 
- [ ] Run E2E tests covering current code and regression
- [ ] All axe checks pass
- [ ] Deploy to Staging behind feature flag

## Acceptance Criteria
- [ ] Complete all unit testing
- [ ] End-to-end tests showing 0 violations
- [ ] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [ ] Deploy changes to Production using feature flag

**Comment by SKasimow** [comment]
@adamwhitlock1  - can you review/point?

**Comment by SKasimow** [comment]
Hi @jcatergovy - take a look at this ticket and lmk if it look like something you can pick up for this sprint.

cc: @adamwhitlock1
